src/kv/store.h

```diff
@@ -757,16 +752,11 @@ namespace kv
           h->append(data.data(), data.size());
           success = DeserialiseSuccess::PASS_SIGNATURE;
         }
-        else if (views.find("ccf.pbft.requests") == views.end())
+        else if (views.find(ccf::Tables::AFT_REQUESTS) == views.end())
         {
-          // we have deserialised an entry that didn't belong to the pbft
-          // requests, nor the pbft new views, nor the pbft pre prepares table
-
-          // NOTE: we currently do not support signature transactions and said
-          // support will be added in the near future
+          // we have deserialised an entry that didn't belong to the bft
+          // requests nor the signatures table
           LOG_FAIL_FMT("Failed to deserialise");
-          LOG_DEBUG_FMT(
```

Comment:
It might be useful to still log the details of the unexpected content, in particular the name of the tables, not just the number of unexpected entries.